src/Integration.Vsix.UnitTests/ErrorList/IssuesSnapshotTests.cs

```diff
@@ -112,8 +114,8 @@ public void IndexOf_SameSnapshotId_ReturnExpectedIndex()
             var revised = original.CreateUpdatedSnapshot("unimportant change");
 
             // Should be able to map issues between two snapshots with the same snapshot id
-            original.IndexOf(999, revised)
-                .Should().Be(999);
+            original.IndexOf(ValidIndexInValidIssueList, revised)
```

Comment:
The product code is now using the supplied index to check whether the issue should be hidden, so the index needs to be valid.